pkg/storaged/clevis-luks-passphrase.sh

```diff
@@ -1,7 +1,13 @@
-#! /bin/bash
+#! /bin/sh
 
 set -eu
 
+type=no
+if [ $# -gt 1 ] && [ "$1" = "--type" ]; then
```

Comment:
This is fine. Just because you were asking about shell yesterday, another common idiom is to use variable defaults `"[ ${1:-}" = "--type" ]`. This gets along with `set -u` without having to explicitly check for existence.